### ISSUE TITLE
Consistency of naming for Loss functions etc NN.Losses.cs

### ISSUE THREAD

**Issue by GeorgeS2019** [xamarin/TorchSharp#307]
[TORCH.NN: **Loss Functions**](https://pytorch.org/docs/stable/nn.html)

## Pytorch
- nn.BCELoss
- nn.BCEWithLogitsLoss
- nn.CosineEmbeddingLoss
- nn.MSELoss
- nn.NLLLoss

## TorchSharp

- bce_loss
- bce_with_logistics_loss
- cosine_embedding_loss
- mse_loss
- nil_loss

This is just a small sample of what I found. Is there a need to be consistent [**more compact naming**] with PyTorch here?

**Comment by GeorgeS2019** [comment]
I suggest more users here report the inconsistencies in naming so eventually, e.g. like **NumpSharp**, we simply **copy and paste** pytorch code in .NET and it works!
